index.bs

```diff
@@ -1266,6 +1278,7 @@ The <dfn method for=StorageManager>getDirectory()</dfn> method steps are:
 
 
 <h2 class=no-num id="acks">Acknowledgments</h2>
+Randell Jesup
```

Comment:
```suggestion

<p>Thanks to Randell Jesup for being awesome!
```
This would match precedent in some other WHATWG standards. https://streams.spec.whatwg.org/#acks has an alternative. Only listing the name seems a little bare?

Reply:
(We could also leave this out for now and do it as a separate PR as there's quite a number of people we should be adding here.)

Reply:
Let's leave it out for now and update it with everyone.  I was just following the guidance for writing a PR :-)